src/backend/BackendInterface.h

```diff
@@ -217,45 +246,40 @@ class BackendInterface
     virtual void
     writeLedger(
         ripple::LedgerInfo const& ledgerInfo,
-        std::string&& ledgerHeader,
-        boost::asio::yield_context& yield) = 0;
+        std::string&& ledgerHeader) = 0;
 
     virtual void
     writeLedgerObject(
         std::string&& key,
-        uint32_t seq,
-        std::string&& blob,
-        boost::asio::yield_context& yield);
+        uint32_t const seq,
+        std::string&& blob);
 
     virtual void
     writeTransaction(
         std::string&& hash,
-        uint32_t seq,
+        uint32_t const seq,
         uint32_t date,
         std::string&& transaction,
-        std::string&& metadata,
-        boost::asio::yield_context& yield) = 0;
+        std::string&& metadata) = 0;
 
     virtual void
-    writeAccountTransactions(
-        std::vector<AccountTransactionsData>&& data,
-        boost::asio::yield_context& yield) = 0;
+    writeAccountTransactions(std::vector<AccountTransactionsData>&& data) = 0;
 
     virtual void
     writeSuccessor(
         std::string&& key,
-        uint32_t seq,
-        std::string&& successor,
-        boost::asio::yield_context& yield) = 0;
+        uint32_t const seq,
+        std::string&& successor) = 0;
+
     // Tell the database we are about to begin writing data for a particular
     // ledger.
     virtual void
-    startWrites(boost::asio::yield_context& yield) const = 0;
+    startWrites() const = 0;
 
     // Tell the database we have finished writing all data for a particular
     // ledger
     bool
-    finishWrites(uint32_t ledgerSequence, boost::asio::yield_context& yield);
```

Comment:
Please change to std::uint32_t const for this instance and elsewhere in this file/PR.